HIRS_ProvisionerTPM2/src/tpm_version.cpp

```diff
@@ -0,0 +1,11 @@
+#include <iostream>
+
+using std::cout;
+using std::endl;
+
+int main(void) {
+    cout << "Chip Version: 2.0.0.0" << endl;
+    cout << "TPM Vendor ID: UNKN" << endl;
```

Comment:
There's really no official way to get the version and vendor from the chip? I could have sworn that we had something like that way back at the start of the TPM 2.0 Provisioner project that Andrew had implemented.

Reply:
I just looked and yeah there is a `tpm2_getcap` tool that could be used to get the info. See the man page, [here](https://github.com/tpm2-software/tpm2-tools/blob/master/man/tpm2_getcap.1.md). That being said it could get tricky as I'm fairly certain that older versions of that tool went under a different name. Likely `tpm2_getcapabilities` or something. That being said, if we wanted to go this route, I'd recommend adding another command to the provisioner to get this information rather than a separate utility binary, like this, since we have a lot of the infrastructure in place for interfacing directly with the TPM there, of course.

Reply:
This was intended only to be a stopgap. We were planning on addressing this in a follow-on issue. In the interest of getting this through, I'd probably be in favor of leaving this as is and addressing it in the subsequent work.

Reply:
tabling this for a new issue